js/webrtc.js

```diff
@@ -511,8 +511,9 @@ var spreedPeerConnectionTable = [];
 				if (mostRecentId !== null) {
 					OCA.SpreedMe.speakers.switchVideoToId(mostRecentId);
 				} else if (enforce === true) {
-					// if there is no mostRecentId is available there is no user left in call
+					// if there is no mostRecentId available, there is no user left in call
 					// remove the remaining dummy container then too
+					OCA.SpreedMe.speakers.unpromoteLatestSpeaker();
```

Comment:
Should have a check if the user that is being removed is the latest speaker?

Reply:
This is already checked a couple of lines above:
https://github.com/nextcloud/spreed/blob/2b4e48c4bc3bf3c65de5b4f215a868e878f2cfaf/js/webrtc.js#L487